src/GraphQL/Utilities/SchemaBuilder.cs

```diff
@@ -178,7 +180,7 @@ private Schema BuildSchemaFrom(GraphQLDocument document)
             }
             else
             {
-                schema.Query = GetType("Query") as IObjectGraphType;
+                schema.Query = (GetType("Query") as IObjectGraphType)!;
```

Comment:
schema.Query = (GetType("Query") as IObjectGraphType)!;
->
 schema.Query = (IObjectGraphType)GetType("Query");

it is easier